src/main/java/uk/gov/ons/census/casesvc/service/CaseService.java

```diff
@@ -28,6 +19,12 @@
   private static final String HOUSEHOLD_INDIVIDUAL_RESPONSE_CASE_TYPE = "HI";
   public static final String CASE_UPDATE_ROUTING_KEY = "event.case.update";
 
+  // At the moment the treatment codes aren't finalised so there's an example one
+  // that tests it. Once the direct delivery treatment codes have been finalised then we can add
+  // them to the list.
+  private static final Set<String> DirectDeliveryTreatmentCodes =
+      new HashSet<>(Arrays.asList("CE_LDIEE"));
```

Comment:
we could put these into `application.yml`. In fact, I think that's what we should do.